test/helper/WebDriver_test.js

```diff
@@ -1172,6 +1172,72 @@ describe('WebDriver', function () {
   });
 });
 
+describe('customLocatorStrategies', function () {
```

Comment:
In case you were wondering How you write/run these tests.

Prerequisites

1. install selenium standalone `npm install selenium-standalone -g` then `selenium-standalone start`
2. install php

then run.

1.`selenium-standalone start` leave running
2. Launch test server: `php -S 0.0.0.0:8000 -t ./test/data/app` leave running
3. Run tests npx mocha "./test/helper/WebDriver_test.js"

You can .only your suite/test to only run  one you want.